test/PlaidML/CppEdsl.DotF16.mlir

```diff
@@ -0,0 +1,61 @@
+// RUN: %python_executable %imex_runner -i %s --pass-pipeline-file=%p/linalg-to-cpu.pp \
+//                                            --runner mlir-cpu-runner -e main \
+//                                            --shared-libs=%mlir_runner_utils \
+//                                            --entry-point-result=void | FileCheck %s
+// RUN: %python_executable %imex_runner -i %s --pass-pipeline-file=%p/linalg-to-llvm.pp \
+//                                            --runner mlir-cpu-runner -e main \
+//                                            --entry-point-result=void \
+//                                            --shared-libs=%mlir_runner_utils,%sycl_runtime | FileCheck %s
+#map0 = affine_map<(d0, d1, d2) -> (d0, d2)>
+#map1 = affine_map<(d0, d1, d2) -> (d2, d1)>
+#map2 = affine_map<(d0, d1, d2) -> (d0, d1)>
+module @dot_f16 {
+  func.func @test(%arg0: tensor<8x16xf16>, %arg1: tensor<16x32xf16>) -> tensor<8x32xf16> {
+    %cst = arith.constant 0.000000e+00 : f16
+    %0 = tensor.empty() : tensor<8x32xf16>
+    %1 = linalg.fill ins(%cst : f16) outs(%0 : tensor<8x32xf16>) -> tensor<8x32xf16>
+<<<<<<< HEAD
```

Comment:
looks like this a merge conflict not resolved

Reply:
Fixed it